### DISCUSSION TITLE
veifying myself

### DISCUSSION THREAD

**Discussion by DipLeChip** [ScratchAddons/ScratchAddons#3207] [Discussion]
yes @helpmepleaseeee on scratch is me @FunctionalMetatable :)

**Comment by apple502j** [comment]
Please log back into your main Scratch account. Logging into an account should cancel the deletion, if within 48 hours. You should also contact Scratch Team.

**Reply by DipLeChip** [reply]
My account was on an old email and I had lots of school work.
On a Friday I left class to go to the toilet and my friends password swipped me and logged onto my Scratch account and set it to deletion. They didn't tell me until Sunday night, so the 48 hours already expired.

Emailed ST about it etc. so hopefully they can recover it or at the least let me recycle the username.
